### ISSUE TITLE
Create way for parents to choose and pass 2 sites to preschool app system

### ISSUE THREAD

**Issue by smarziano** [smartchicago/chicago-early-learning#742]
Currently, parents can pass 6 sites' information from Chicago Early Learning to the ECM application system (see issue #655 & #678). In the upcoming phase, we want to only pass two sites' information to the preschool application system. However, we want parents to have an option to view and compare as many sites as they want. 

Let's create a way on this Compare & Apply (/starred) page for parents to "check" the sites they choose to pass. How I'm imagining it now is that there is a checkbox at the top of each site's that parents would check. Upon checking it would move the site to an area near the top with the "Apply" button next to the top two choices.

@theandrewbriggs I know we spoke about this, let me know if you think it would be helpful if I wireframe this page.

**Comment by smarziano** [comment]
We still need text on this page to help parents understand that they should choose only 2 sites to apply to. This should be addressed with upcoming copy edits and is a separate issue.

Therefore, I'm saying that this issue is "waiting on merge."

**Comment by smarziano** [comment]
Two thoughts that came up as I was reviewing this today:
1) Do we need parents to select via Chicago Early Learning which is their first choice, and which is their second? Or can they decide that in the preschool application?
2) On mobile, can we make it clearer if parents try to click more than 2 that they can't? We have text at the top, but I would like to think of a better solution.

I will create new issues for those questions and close this one.

Also, I want to call @theandrewbriggs out for doing such a great job on this Compare & Apply page.
